src/lib/serviceRoutinesV2/serviceRoutinesCommon.cpp

```diff
@@ -39,27 +39,20 @@
 
 /* ****************************************************************************
 *
-* setFilters -
```

Comment:
Old setFilters() function has been split into two ones: one for attributes and one for metadata.

Reply:
NTC (informative)